lighthouse-core/test/computed/metrics/total-blocking-time-test.js

```diff
@@ -72,27 +63,21 @@ Object {
       const fcpTimeMs = 1500;
       const interactiveTimeMs = 2500;
 
-      expect(CumulativeLongQueuingDelay.calculateSumOfLongQueuingDelay(
-        events,
-        fcpTimeMs,
-        interactiveTimeMs
-      )).toBe(150);
+      expect(
+        TotalBlockingTime.calculateSumOfBlockingTime(events, fcpTimeMs, interactiveTimeMs)
+      ).toBe(150);
     });
 
-    it('clips queuing delay regions properly', () => {
-      const fcpTimeMs = 1050;
-      const interactiveTimeMs = 2050;
+    it('treats the end of task as blocking time', () => {
+      const fcpTimeMs = 100;
+      const interactiveTimeMs = 200;
+      // The last 50ms of this 100ms task is blocking time, and it happens after FCP, so it's
```

Comment:
oh this is unexpected to me, there's no clipping involved at all? I would expect start of task to be clipped to `FCP` and so the "last 50" would still be free